src/test/java/NoJUnitTest.java

```diff
@@ -0,0 +1,61 @@
+import dev.nolij.zson.Zson;
+import dev.nolij.zson.ZsonValue;
+
+import java.util.Map;
+
+import static dev.nolij.zson.Zson.*;
+
+public final class NoJUnitTest {
```

Comment:
Why do we need this?

Reply:
honestly I kind of forgot but I wanted to test some stuff outside of gradle
it could be removed

Reply:
Let's convert any new tests in it to JUnit.